bedrock/etc/redirects.py

```diff
@@ -0,0 +1,3 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
```

Comment:
Don't really need this file. We can always add it later should we grow some redirects for these. The only required potentially empty file for an app is `models.py`.